GabrielaRamirez/EjerciciosHackerrank/src/basics/InheritanceII.java

```diff
@@ -0,0 +1,29 @@
+package basics;
+
+    class Arithmetic{
+
+        int add(int x, int y){
```

Comment:
Remember, variable names and parameter names must be meaningful:
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names

Reply:
ok, i'll change